src/main/java/com/exadel/discountwebapp/statistics/controller/StatisticsController.java

```diff
@@ -0,0 +1,33 @@
+package com.exadel.discountwebapp.statistics.controller;
+
+import com.exadel.discountwebapp.statistics.dto.RequestDataDTO;
+import com.exadel.discountwebapp.statistics.dto.SummaryStatisticsDTO;
+import com.exadel.discountwebapp.statistics.service.StatisticsService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.*;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+
+@RequiredArgsConstructor
+@RestController
+@RequestMapping("/api/statistics")
+public class StatisticsController {
+
+    private final StatisticsService statisticsService;
+
+    @PreAuthorize("hasAuthority('ADMIN')")
+    @PostMapping()
```

Comment:
We use GET method here and pass all needed attributes as request params

Reply:
fixed